addons/phone_validation/tests/test_phonenumbers_patch.py

```diff
@@ -51,3 +51,27 @@ def test_region_PA_monkey_patch(self):
         parsed = phonenumbers.parse('+507 833 8744')
         self.assertTrue(phonenumbers.is_valid_number(parsed))
         self.assertEqual(parsed.national_number, 8338744)
+
+    def test_region_MU_monkey_patch(self):
```

Comment:
:socks: should be defined before "test_region_PA_monkey_patch" (alphabetical ordering so that it matches test execution order)